exercises/wordy/src/example/java/WordProblemSolver.java

```diff
@@ -0,0 +1,60 @@
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+final class WordProblemSolver {
+
+    private static final String INTEGER_REGEX_STRING = "(-?\\d+)";
+
+    private static final String BINARY_OPERATION_REGEX_STRING
+        = "(plus|minus|multiplied by|divided by|raised to the power)";
+
+    private static final String VALID_QUESTION_REGEX_STRING =
```

Comment:
insert obligatory line about regexes... like if you've got a problem then try to solve it with a regex, now you got two problems. :)

It's great mental push-ups, but likely too dense to be maintainable.